mc/Mongo-Core.package/Mongo.class/instance/query..st

```diff
@@ -5,6 +5,6 @@ query: aMongoQuery
 	aMongoQuery doBlock isNil ifTrue: [
 		aMongoQuery doBlock: [:each | results add: each]].
 	(MongoCursor root: self query: aMongoQuery ) execute.
-	(results isNotEmpty and: [ results first includesKey: '$err' ])
+	(results isNotEmpty and: [ (results first includesKey: '$err') or: [ (results first includesKey: 'errmsg') and: [ (results first includesKey: 'code') and: [ results first includesKey: 'ok' ] ] ] ])
```

Comment:
I'd like to know more about the new check (especially the 2 "and"s). I will write in words what I read in the code:

*Before*: We detected an error when there is a `$err` key.
*Now*: Alternatively, we will also detect an error when there are 3 keys simultaneously `errmsg`, `code` and `ok`.

Would you have some URL that documents this detection?
As a matter of fact, I don't find at this moment other info about $err than a mention [here](https://docs.mongodb.com/v4.2/reference/mongodb-wire-protocol/) in the Wire Protocol docs.